change-notes/1.26/analysis-cpp.md

```diff
@@ -8,6 +8,7 @@ The following changes in version 1.26 affect C/C++ analysis in all applications.
 
 | **Query**                   | **Tags**  | **Purpose**                                                        |
 |-----------------------------|-----------|--------------------------------------------------------------------|
+| Unsafe use of this-pointer (`cpp/Likely Bugs/OO/UnsafeUseOfThis.ql`) | correctness, language-features, security | This query finds pure virtual function calls whose qualifier is an object under construction. Results are shown on LGTM by default. |
```

Comment:
We no longer update this file. New change notes go in https://github.com/github/codeql/tree/main/cpp/change-notes

Reply:
Oh right! 🤦 I completely forgot about that one. Will fix!

Reply:
Fixed in b3400c4f1556b624fa11ad371c4591b474d0655d.